src/map/location.hpp

```diff
@@ -27,6 +27,10 @@ class variable_set;
 
 struct wml_loc {};
 
+struct hex_location {
```

Comment:
Needs documentation

Reply:
The name is bad, given that `hex` and `location` are both heavily used in this area of the code. `cubic_location` would be a bit better.

Still needs better documentation - or a link to good documentation. Saying that it's "also called cubic coordinates" isn't that helpful, because searching for that on the web returns a lot of pages talking about using three orthogonal axis for three-dimensional space, which then takes time to find the ones talking about using a 2d plane within that space.

Remember, even if it's the first result that you see when you do a web search, that can be because the search engine is using your browsing history.

Reply:
Sure, I can add a link to redblobgames.com.